e2e/cypress/integration/emoji/react_to_last_message_shortcut_spec.js

```diff
@@ -0,0 +1,575 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+import * as MESSAGES from '../../fixtures/messages';
+import users from '../../fixtures/users.json';
+
+const newChannelName = `channel-react-to-last-message-${Date.now()}`;
+let channelId = '';
+let newChannel = {};
+
+describe('React to last message shortcut', () => {
+    before(() => {
+        cy.apiLogin('sysadmin');
+        cy.visit('/');
+
+        // # Get the current channels Id for later use such as posting message with other user
+        cy.getCurrentChannelId().then((id) => {
+            channelId = id;
+        });
+
+        // # Create a new channel for later use such as when channel is empty test
+        cy.getCurrentTeamId().then((teamId) => {
+            // eslint-disable-next-line no-magic-numbers
+            cy.apiCreateChannel(teamId, newChannelName, newChannelName).then((response) => {
+                newChannel = Object.assign({}, response.body);
```

Comment:
Please run all the test against this new channel. I suggests to visit it on `beforeEach` block.

Reply:
Okay, nvm this comment.  I thought you'll be testing all against the new channel which is not the case.  This new channel is necessary for the last test only "Should not open emoji picker by shortcut if last post is a system message".